### PR TITLE
Fix: update safe-deployments

### PR DESCRIPTION
**Pull request by katspaugh**
## What it solves

The `safe-deployments` package was migrated to a new org + the latest version adds a new network.

### CODE REVIEW

File: packages/safe-core-sdk-utils/package.json [hunk 1]
```diff
@@ -31,7 +31,6 @@
   ],
   "homepage": "https://github.com/safe-global/safe-core-sdk#readme",
   "devDependencies": {
-    "@gnosis.pm/safe-deployments": "1.19.0",
```
Comment (katspaugh) [inline packages/safe-core-sdk-utils/package.json:34]: Does it really need to be only a dev dependency?

**Comment by germartinez** [comment]
The `networks` object need to be updated here: `safe-core-sdk/packages/safe-core-sdk-utils/src/eip-3770/config.ts`

File: packages/safe-core-sdk-utils/package.json (hunk 1 above)
Comment (germartinez) [reply packages/safe-core-sdk-utils/package.json:34]: Inside the `safe-core-sdk-utils` package, yes

**Comment by germartinez** [comment]
@katspaugh These chainIds should be added as well: 108, 4689, 9728, 43288, 56288

**Review by germartinez — APPROVED** [review]

**Pull request merged**